homeassistant/components/unifi/device_tracker.py

```diff
@@ -138,6 +139,30 @@ def update_controller():
 
     async_dispatcher_connect(hass, controller.event_update, update_controller)
 
+    @callback
+    def update_disable_on_entities():
```

Comment:
So the downside of this approach is that changing one option will re-apply all other config options.

Reply:
How would you prefer it to be done?

Reply:
Well I was thinking, would it be possible to see if just wired clients change, we just update those?

Right now you might customize a bunch, disable devices via option and all other customization is lost

Reply:
Should I maybe not override disabled by user?

Reply:
I think that it should be fine to just add your own values for `disabled_by`, and allow them to be translatable.

Reply:
Im reverting any smart updating of disabled by. Better to get this integrated first and then I can revisit and find out a proper way of updating only relevant entities